setup/questions.sh

```diff
@@ -193,6 +193,16 @@ if [ -z "${STORAGE_ROOT:-}" ]; then
 	STORAGE_ROOT=$([[ -z "${DEFAULT_STORAGE_ROOT:-}" ]] && echo "/home/$STORAGE_USER" || echo "$DEFAULT_STORAGE_ROOT")
 fi
 
+# Check if Nextcloud is already installed
+# If it isn't, ask the user if Nextcloud should be disabled 
+if [ ! -d $STORAGE_ROOT/owncloud ]; then
+	# Ask the user if he/she wants to disable Nextcloud
+	yesno_box "Disable Nextcloud?"\
+		"Nextcloud offers the Contacts and Calendar features in Mail-in-a-box\
```

Comment:
I think this description needs some more work, by default MIAB uses Nextcloud for other things as well (for example file sharing) so perhaps these should be mentioned here.

I don't know whether it should be explained to the user what the consequences of disabling it would mean too (especially if they have data stored using Nextcloud).

Reply:
> I think this description needs some more work, by default MIAB uses Nextcloud for other things as well (for example file sharing) so perhaps these should be mentioned here.

Fixed.

> I don't know whether it should be explained to the user what the consequences of disabling it would mean too (especially if they have data stored using Nextcloud).

I don't think that the user would see this message at all if he already has Nextcloud installed, since the script checks if the Nextcloud folder doesn't exist before asking the question.

Reply:
You are right.

How would a user disable Nextcloud if the user had already installed it on a previous installation attempt?

Reply:
I think that removing an existing Nextcloud installation is a fairly easy process (stop  the related services, remove the related packages, edit the related configs, etc...).
Should I put that in a separate pull request or put it in this one as well?

Reply:
My thoughts would be to include it with this one as it is so closely related.  Perhaps @JoshData might be able to comment on what he would rather have?

Reply:
Alright, while waiting for @JoshData to comment on this, I will start writing a script for removing Nextcloud.

Reply:
This should be removed and replaced by simply reading an environment variable for now. I don't want to ask any unnecessary configuration questions to the end user.